crates/rspack_loader_swc/src/lib.rs

```diff
@@ -0,0 +1,221 @@
+#![feature(let_chains)]
+
+use std::default::Default;
+use std::sync::Arc;
+
+use rspack_core::{rspack_sources::SourceMap, LoaderRunnerContext, Mode};
+use rspack_error::{errors_to_diagnostics, internal_error, Error, Result};
+use rspack_loader_runner::{Identifiable, Identifier, Loader, LoaderContext};
+use serde::Deserialize;
+use swc_config::config_types::{BoolConfig, MergingOption};
+use swc_config::merge::Merge;
+use swc_core::base::config::{
+  Config, ErrorConfig, FileMatcher, InputSourceMap, IsModule, JscConfig, ModuleConfig, Options,
+  SourceMapsConfig, TransformConfig,
+};
+use swc_core::base::{try_with_handler, Compiler};
+use swc_core::common::comments::SingleThreadedComments;
+use swc_core::common::{FileName, FilePathMapping, GLOBALS};
+use swc_core::ecma::transforms::base::pass::noop;
+
+#[derive(Debug, Default, Deserialize)]
+#[serde(rename_all = "camelCase", default)]
+pub struct SwcLoaderJsOptions {
```

Comment:
Is this option align with `swc`? If it is, could we just use the option of `swc`?